DragonLoopAPI/Managers/BusHistoryManager.cs

```diff
@@ -0,0 +1,28 @@
+using DragonLoopAPI.Models;
+using DragonLoopModels;
+using System.Threading.Tasks;
+
+namespace DragonLoopAPI.Managers
+{
+    public class BusHistoryManager
+    {
+        private readonly DragonLoopContext _context;
+
+        public BusHistoryManager(DragonLoopContext context)
+        {
+            _context = context;
+        }
+
+        public async Task<BusHistory> AddHistory(Bus bus)
```

Comment:
This method doesn't need to be async.

Also, you need to add BusHistory to the DragonLoopContext. You can follow what I did for all of the other models. Then once you create this new BusHistory object, you can add it to the context list like I did for adding new buses.

Again though, I feel like we can do this with a database trigger, but if we want to do it in the API, you need to add it to the DB context.